src/core/Aggregators.java

```diff
@@ -34,7 +33,7 @@
     LERP,   /* Regular linear interpolation */
     ZIM,    /* Returns 0 when a data point is missing */
     MAX,    /* Returns the <type>.MaxValue when a data point is missing */
-    MIN     /* Returns the <type>.MinValue when a data point is missing */
+    MIN,    /* Returns the <type>.MinValue when a data point is missing */
```

Comment:
https://github.com/OpenTSDB/opentsdb/blob/next/src/core/Aggregators.java

Seems like the codes in class "First" and class "Last"  are almost the same?